specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/blob.json

```diff
@@ -993,6 +993,14 @@
         "immutableStorageWithVersioning": {
           "$ref": "#/definitions/ImmutableStorageWithVersioning",
           "description": "The object level immutability property of the container. The property is immutable and can only be set to true at the container creation time. Existing containers must undergo a migration process."
+        },
+        "enableNfsV3RootSquash": {
+          "type": "boolean",
```

Comment:
It seems the feature still not work in my subscription in canary.
Do I need to enable any feature flag to run the feature?

```
PUT https://management.azure.com/subscriptions/{subcriptionID}/resourceGroups/res6672/providers/Microsoft.Storage/storageAccounts/sto2847/blobServices/default/containers/container2042?api-version=2021-06-01 HTTP/1.1
x-ms-client-request-id: 232013b6-fc21-4eac-8b5a-3072da1c5280
Accept-Language: en-US
Authorization: Bearer ***
User-Agent: FxVersion/4.6.30321.06 OSName/Windows OSVersion/Microsoft.Windows.10.0.19043. Microsoft.Azure.Management.Storage.StorageManagementClient/22.0.0.0
Content-Type: application/json; charset=utf-8
Content-Length: 63
Host: management.azure.com

{
  "properties": {
    "enableNfsV3RootSquash": true
  }
}
HTTP/1.1 400 Bad Request
Cache-Control: no-cache
Pragma: no-cache
Content-Length: 118
Content-Type: application/json
Expires: -1
x-ms-request-id: 610f3434-d8fe-47cc-84c0-e56f7558d6d4
Strict-Transport-Security: max-age=31536000; includeSubDomains
Server: Microsoft-Azure-Storage-Resource-Provider/1.0,Microsoft-HTTPAPI/2.0 Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-writes: 1197
x-ms-correlation-request-id: 01f680f4-ea12-4454-bd69-7b85e4fe4897
x-ms-routing-request-id: SOUTHEASTASIA:20210816T094055Z:01f680f4-ea12-4454-bd69-7b85e4fe4897
X-Content-Type-Options: nosniff
Date: Mon, 16 Aug 2021 09:40:54 GMT

{"error":{"code":"FeatureNotSupportedForAccount","message":"EnableNfsV3RootSquash is not supported for the account."}}
```